ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/EthGetMinerDataByBlockHash.java

```diff
@@ -89,10 +89,11 @@ public static MinerDataResult createMinerDataResult(
                         .transactionReceiptByTransactionHash(t.getTransaction().getHash())
                         .map(
                             receipt ->
-                                receipt
-                                    .getTransaction()
-                                    .calcEffectiveGas(receipt.getBaseFee())
-                                    .multiply(receipt.getGasUsed()))
+                                Wei.of(
+                                    receipt
+                                            .getTransaction()
+                                            .getEffectivePriorityFeePerGas(receipt.getBaseFee())
```

Comment:
AFAIK - this should be effective gas, not effective priority fee

Reply:
:man_facepalming: good point